joomla/joomla-cms
# Remove test producing unpredictable results

Our Javascript tests and our setup for automated testing is quite unstable right now. The main reason are the in this PR removed test.

### Summary of Changes
Removed the joomla-field-send-test-mail test


### Testing Instructions
Let the javascript test running and check the results (see https://docs.joomla.org/Running_JavaScript_Tests_for_the_Joomla_CMS)

### Expected result
no failures


### Actual result
failes from time to time
